00_download_data/nbconverted/1B_preprocess_27k_methylation_data.py

```diff
@@ -217,9 +218,77 @@ def impute_leq(methylation_df, n_na):
 tcga_methylation_df.to_csv(filtered_file, sep='\t', float_format='%.3g')
 
 
+# ### Read TCGA Barcode Curation Information
+# 
+# Extract information from TCGA barcodes - `cancer-type` and `sample-type`. See https://github.com/cognoma/cancer-data for more details
+
+# In[11]:
+
+
+# commit from https://github.com/cognoma/cancer-data/
+sample_commit = 'da832c5edc1ca4d3f665b038d15b19fced724f4c'
+url = 'https://raw.githubusercontent.com/cognoma/cancer-data/{}/mapping/tcga_cancertype_codes.csv'.format(sample_commit)
+cancer_types_df = pd.read_csv(url,
+                              dtype='str',
+                              keep_default_na=False)
```

Comment:
Referencing data from a specific commit is definitely a good idea. Does it make sense to have the commit live in the config file in case you want to change versions later?

Reply:
Especially since you use the commit in multiple notebooks

Reply:
Yeah, good catch - I copy/pasted this code but the config file seems like a much better idea